crds/jwst/specs/combined_specs.json

```diff
@@ -5131,6 +5131,100 @@
             "tpn":"niriss_saturation.tpn",
             "unique_rowkeys":null
         },
+        "speckernel":{
```

Comment:
Sounds like some kind of rare songbird

Reply:
Possibly some bird sang about by Paul McCartney...